src/main/cpp/profiler/init_nvtxw.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Licensed under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.txt for license information.
+ */
+
+#pragma once
+
+#include "NvtxwEvents.h"
+#include "nvtxw3.h"
+
+#include <filesystem>
+#include <optional>
+#include <string>
+
+namespace spark_rapids_jni::profiler {
+
+bool createNvtxwStream(const nvtxwInterfaceCore_t* nvtxwInterface,
+                       const nvtxwSessionHandle_t& session,
+                       const std::string& name,
+                       const std::string& domain,
+                       nvtxwStreamHandle_t& stream);
+
```

Comment:
No external use thus I would recommend to delete it.
```suggestion
```

Reply:
It is used once in `spark_rapids_profile_converter.cpp`, line 884